### PR TITLE
Feature/dashboard styling

### PR DESCRIPTION
**Pull request by DrewBradley**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Adds two elements to the dashboard:
1. Preview of the current tour
2. Short cut to top contacts

## *Instructions for reviewer:*
*Where should the reviewer start?*
`src/components/dashboard/ContactsPreview.js`
`src/components/dashboard/Dashboard.css`
`src/components/dashboard/Dashboard.js`
`src/components/dashboard/TourDisplay.js`
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
Screen Shot 2021-04-15 at 5 25 50 PM

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/dashboard/Dashboard.js [hunk 1]
```diff
@@ -3,12 +3,15 @@ import './Dashboard.css';
...
 import listPlugin from '@fullcalendar/list'
 import {ListWrapper} from './ListWrapper'
+import TourDisplay from './TourDisplay'
+import ContactsPreview from './ContactsPreview'
 
 
 const Dashboard = () => {
   return (
     <div className="Dashboard">
-      <h1>THIS IS THE Dashboard</h1>
```
Comment (kevinhartmann23) [inline src/components/dashboard/Dashboard.js:11]: Nice! Got rid of the "this is dashboard" 😂

**Review by kevinhartmann23 — APPROVED** [review]
Great work drew, thank you!!

**Review by richardltyler — APPROVED** [review]
Thanks for all of this! I love the contacts display

File: src/components/dashboard/Dashboard.js (hunk 1 above)
Comment (DrewBradley) [reply src/components/dashboard/Dashboard.js:11]: How will we know?

**Review by kristenmb — APPROVED** [review]
This is great! Thanks

**Pull request merged**